rust_snuba/src/strategies/python.rs

```diff
@@ -282,7 +282,7 @@ mod tests {
     use rust_arroyo::testutils::TestStrategy;
 
     fn set_sys_executable() {
-        let python_executable = std::env::var("PYTHONEXECUTABLE").unwrap();
+        let python_executable = std::env::var("SNUBA_PYTHONEXECUTABLE").unwrap();
```

Comment:
apparently PYTHONEXECUTABLE is already a used envvar name for some pip (or python? or virtualenv?) internals